### ISSUE TITLE
color picker: can't type in or delete hex value due to autocomplete

### ISSUE THREAD

**Issue by emailjough** [elementor/elementor#10030]
## Prerequisites

- [ x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x ] The issue still exists against the latest stable version of Elementor.

## Description

1. When attempting to type a new hex value into the color picker in any widget, column or section, auto complete replaces with #000000

2. With cursor at end of hex value, cant backspace to delete existing value in order to type in new one. Auto complete replaces with #000000

## Steps to reproduce

Step 1: open color picker for the first time in any widget, column or section
Step 2: attempt to type a hex code into the field, either with or without a #
Step 3: notice that a 6-digit hex is immediately inserted into the field with your insertion point placed at the end of it, and the characters you are currently typing are appended to the end resulting in some random variation or gradation of that color.
Step 4: use the clear button to clear the unwanted color, then reenter your color, which this time works as expected.
Step 5: click outside the color picker to close it
Step 6: open the color picker again and repeat Step 2 and 3

Other Related issues:
Step 7: use the backspace key to delete an existing hex color and notice how some (seemingly) random color gets added to the widget, column or section despite none being selected in the color picker.
Step 8: choose a color from the saved swatches at the top of the color picker and notice how no hex number is added to the field, thus giving no way to delete or edit the color. 
Step 9: close the color picker and reopen, attempt to use backspace to delete the existing color and notice how auto complete again inserts a 6 digit hex, and prevents you from deleting it.

For me (using Chrome), this is consistently repeatable by simple closing and reopening the color picker.

In short, this color picker is a ridiculous mess. The UI is terrible, practically unusable.  

 
## Isolating the problem

- [x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ x] This bug happens with a default WordPress theme active.
- [x ] I can reproduce this bug consistently using the steps above.

**Comment by shilo-ey** [comment]
Please modify your Steps to reproduce to a proper one.

**Comment by black-eye** [comment]
+1

**Comment by emailjough** [comment]
> Please modify your Steps to reproduce to a proper one.

Steps updated

**Comment by joshuamayo** [comment]
This hasn't started happening for me until this new update to the color picker. For example, I begin typing in my hex code, but then the field where you type in the hex code auto-fills with some random code. For example, I want to type in hex code #F7F7F7, so I begin typing, but by the time I reach the second letter... F7... it auto-fills the code as #FF77FF. This plus the other issues @emailjough described is what I'm experiencing.

**Comment by MichaelDarko** [comment]
Yes, this is a nightmare. It seems like it started happening after latest update. No idea what is the purpose of the autocomplete. It makes it impossible to type-in any value by hand.

If you're copying the whole HEX value, you don't need autocomplete. If you're typing the value by hand, you don't want autocomplete. How can any autocomplete predict what i want to type?

The new color picker is a bit of a mess, to be honest.

- Autocomplete means you can't type any value by hand
- Transparency is not in rgba format, so you can't figure out the transparency % value
- And i'm still experiencing bugs where selected color will not apply on the element

**Comment by emailjough** [comment]
I have also experienced the selected color not applying to the element. When this happens I have to use the transparency slider, sliding it slightly to the left and then back all the way to the right. Then the color applies.  Ridiculous. 

> * And i'm still experiencing bugs where selected color will not apply on the element

**Comment by Webtica** [comment]
+1 same here

**Comment by realstephenzhao** [comment]
Bad update, they even bring 8 digit #rgba hex color too.

**Comment by zeinnicholas** [comment]
@shilo-ey are you going to ignore the previous issue about this and other problems?

https://github.com/elementor/elementor/issues/9862

**Comment by brunov99** [comment]
Same for me.
To enter different color and bypass autocomplet I need to click on clear button first and then I can type a new color value and sometimes I have to type the # character after.

**Comment by aidan2496** [comment]
+1 For this issue
